src/utils/azureClientUtil.ts

```diff
@@ -15,18 +15,9 @@ export namespace azureClientUtil {
 
     // tslint:disable: no-unsafe-any
     export async function selectSubscription(context: IActionContext): Promise<string> {
-        const azureAccountExtension = vscode.extensions.getExtension('ms-vscode.azure-account');
-        // tslint:disable-next-line: no-non-null-assertion
-        const azureAccount = azureAccountExtension!.exports;
-        await azureAccount.waitForFilters();
-        if (azureAccount.status !== 'LoggedIn') {
-            throw new Error(localize("", "Please Log in at first!"));
-        }
-        const subscriptions : {id: string, name: string}[] = azureAccount.filters.map(filter => {return {id: filter.subscription.subscriptionId, name: filter.subscription.displayName}; });
-        const subscriptionId = await context.ui.showQuickPick(subscriptions.map((s) => {
-            const option = s.id.concat(' (', s.name, ')');
-            return { label: option, subscriptionId: s.id};
-        }),                                               { canPickMany: false, placeHolder: localize("", "Please choose the Azure subscription")});
-        return subscriptionId.subscriptionId;
+        context;
```

Comment:
remove this line?

Reply:
There is a eslint rule, according to the function, we need to pass context, but if the context not in use, there will get build error.

Reply:
I remember we can add `_` to the beginning of the variable name as `_context`. Then there will be no lint error. Can we have a try?